test/unit/s3.test.ts

```diff
@@ -6,29 +6,33 @@ import 'dotenv/config';
 
 const _test = tapePromise(test);
 
-const s3 = new S3(process.env.BASE_URL_TEST, {
-    bucketName: process.env.BUCKET_NAME_TEST,
-    keyId: process.env.KEY_ID_TEST,
-    secretAccessKey: process.env.SECRET_ACCESS_KEY_TEST,
-});
+_test('S3 checks', async(t) => {
+    if (process.env.BUCKET_NAME_TEST) {
```

Comment:
IMO would nicer to follow `return early` principle here

Reply:
Done.